larray/core/array.py

```diff
@@ -5681,6 +5690,15 @@ def to_excel(self, filepath=None, sheet_name=None, position='A1', overwrite_file
         >>> # add to existing sheet starting at position A15
         >>> a.to_excel('test.xlsx', 'Sheet1', 'A15')  # doctest: +SKIP
         """
+        if isinstance(sheet_name, Group):
```

Comment:
I would rather use `sheet_name = str(_to_tick(sheet_name))` instead of all this code